test/integration/client-side-encryption/client_side_encryption.prose.corpus.test.js

```diff
@@ -214,10 +215,12 @@ describe('Client Side Encryption Prose Corpus Test', function () {
               tlsCertificateKeyFile: process.env.KMIP_TLS_CERT_FILE
             }
           };
+          const extraOptions = getEncryptExtraOptions();
           const autoEncryption = {
             keyVaultNamespace,
             kmsProviders,
-            tlsOptions
+            tlsOptions,
+            extraOptions
```

Comment:
All these tests now include `extraOptions.sharedCryptLibPath` when `CRYPT_SHARED_LIB_PATH` is in the env.